logger/options.go

```diff
@@ -104,10 +105,13 @@ func (options *Options) setDefaults() {
 	if options.FlushInterval == 0 {
 		options.FlushInterval = defaultFlushInterval
 	}
+	if options.IngestURL == "" {
+		options.IngestURL = defaultIngestURL
+	}
 	if options.MaxBufferLen == 0 {
 		options.MaxBufferLen = defaultMaxBufferLen
 	}
-	if options.IngestURL == "" {
-		options.IngestURL = defaultIngestURL
+	if options.Timestamp == 0 {
```

Comment:
i dont think we'll want a default here - it doesn't make sense to have a Logger-level timestamp set. 

we shouldn't need to do anything with `Timestamp` until we assemble the `Line`:

https://github.com/logdna/logdna-go/blob/aa78e9675d21a907d1ee7078c4aaae7d48411836/logger/transport.go#L94-L99

that would change to something like this:

```golang
line := Line{
  Body:  msg.Body,
  App:   msg.Options.App,
  Env:   msg.Options.Env,
  Level: msg.Options.Level,
}

if msg.Options.Timestamp.isZero() {
  line.Timestamp = time.Now().UnixNano() / int64(time.Millisecond)
} else {
  line.Timestamp = msg.Options.Timestamp.UnixNano() / int64(time.Millisecond)
}
```